alienfile

```diff
@@ -0,0 +1,41 @@
+use alienfile;
+use Config;
+
+configure { requires 'Alien::Build::Plugin::Build::Make' => '0.01' };
+
+plugin 'Probe::Vcpkg' => (
+    name     => 'libsodium',
+    ffi_name => 'libsodium',
+);
+plugin 'PkgConfig' => (
+    pkg_name => 'libsodium',
+    minimum_version => '1.0.3',
+);
+
+share {
+    my $run_upstream_tests = ($ENV{AUTOMATED_TESTING} || $ENV{RELEASE_TESTING}) ? 1 : 0;
+
+    # note on apple weirdisms: https://github.com/joyent/libuv/issues/1200
+    meta->prop->{env}->{LIBTOOLIZE} = 'libtoolize' if $^O eq 'darwin';
```

Comment:
Was this needed generally for macOS? I thought this was a weird-ism of libuv.

Reply:
You're right. My glance at a reading through the ticket made me think it was a generic MacOS issue